port/zos390/omr__le_api.h

```diff
@@ -128,6 +128,6 @@ typedef struct j9_31bit_mch {
 	uint32_t padding6[32];
 
 	U_128 vr[32];
-} j9_31bit_mch;
+} omr_31bit_mch;
```

Comment:
Does this need a typedef similar to J9ZOSLEConditionHandlerRecord?

Reply:
This struct is only used in OMR. Internal J9 zOS builds didn't complain about missing `j9_31bit_mch`.